ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/BackingStore/InMemory.hs

```diff
@@ -0,0 +1,165 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE LambdaCase         #-}
+{-# LANGUAGE RankNTypes         #-}
+
+-- | An implementation of a backing store using a TVar. This is the
```

Comment:
Maybe we could link to the backing store module here.

Reply:
Done